tutorials/animation/introduction_2d.rst

```diff
@@ -66,7 +65,8 @@ Computer animation relies on keyframes
 
 A keyframe defines the value of a property at a certain point in time.
 
-White and blue diamond shapes represent keyframes in the timeline.
+White and blue diamond shapes represent keyframes in the timeline. A
```

Comment:
This might be misleading as there is no further explanation on why white and blue and if there is a difference between the colors.

I assume what the original author meant was just that a keyframe icon turns blue when selected, however Godot 3.1's bezier curve keyframes are actually blue (and call method keyframes were green even before...).

Since they all behave the same maybe drop the "White and blue" part entirely?